src/node.cc

```diff
@@ -259,28 +259,32 @@ static v8::Isolate* node_isolate;
 
 node::DebugOptions debug_options;
 
+#ifdef NODE_USE_V8_PLATFORM
+NodePlatform* NodePlatform::platform = nullptr;
```

Comment:
As far as I can tell, we don't actually need to switch this to a static field. It's initialized the same way in the `Initialize()` method, it's just pointing at `NodePlatform::platform` rather than `_platform`. It doesn't seem like `NodePlatform` itself actually makes any use of that reference. Did I miss something?

Reply:
Updated with a new approach :)